LectureNotes/Week-04/readme.md

```diff
@@ -0,0 +1,12 @@
+# Week 4 - Password Validation and Salted Hashing
+### Application Security And Hardening
+
+[Return Home](../../../../)  
+
+[View Lecture Notes](http://coreyshuman.github.io/GeekwiseApplicationSecurity/LectureNotes/Week-04)  
+
+### Goals
+
+### Topics
+
+### Applications
```

Comment:
Applications:

- [Week 04 - Insecure Blog App Part 3](../../Applications/Week-04/01-BasicBlogAppPart03)
  - A work-in-progress blogging application with basic user authentication and authorization

Reply:
Also add a link to this application at the end of the html notes. Use the same content :)